CalcualtorLogic.js

```diff
@@ -38,65 +38,15 @@ class Calculator {
         const row1 = this.getButtonsRow(["7", "8", "9"]);
         const row2 = this.getButtonsRow(["4", "5", "6"]);
         const row3 = this.getButtonsRow(["1", "2", "3"]);
+        const row4 = this.getButtonsRow(["0", ".", "±"]);
```

Comment:
this is wrong.
I said only for  * + - 

also you can't use `getButtonsRow` method for that 
because `getButtonsRow` will set data-number attribute 
while the operators buttons need `data-operator` attribute 


**you need to create a new method similar to getButtonsRow**

Reply:
Already firstly I created
getMlutyAddSubtractRow
later I changed the name to :
getButtonsRow2
getButtonsRow3 
 And it came in my smart brain(mind) these buttons are operators and should deal with them not as numbers 

but I forgot to change the data name
and forgot to change the method name 
in this line:
 const row4 = this.getButtonsRow(["0", ".", "±"]);

so it was working 
later I deleted the both new methods